repair_type/models/repair.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Patryk Pyczko (APSL-Nagarro)<ppyczko@apsl.net>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class RepairOrder(models.Model):
+    _inherit = "repair.order"
+
+    product_location_dest_id = fields.Many2one(
```

Comment:
I agree overall with the change :+1: 

However, being OCA we need to be as modular as possible. Since this fields doesn't do anything without a future `repair_picking_after_done` extension, I think we would need to move that to a glue module: something like `repair_type_picking_after_done` where the field is added and affects the picking after done behavior.

Does this sound reasonable?

Reply:
Thank you for your feedback @LoisRForgeFlow, your suggestion makes a lot of sense. 

I’ve applied the suggested changes by removing the fields related to the repaired product's location out of repair_type and will move them into a separate module as suggested. Let me know if there’s anything else to adjust!